packages/shared/src/components/ExtensionOnboarding.tsx

```diff
@@ -0,0 +1,43 @@
+import React, { ReactElement } from 'react';
+import Link from 'next/link';
+import Logo, { LogoPosition } from './Logo';
+import { OnboardingTitleGradient } from './onboarding/common';
+import { onboardingUrl } from '../lib/constants';
+import { Button, ButtonSize } from './buttons/Button';
+import { cloudinary } from '../lib/image';
+
+const ExtensionOnboarding = (): ReactElement => {
+  return (
+    <div className="flex overflow-hidden flex-col justify-center items-center px-7 antialiased text-center min-h-[100vh] max-h-[100vh]">
```

Comment:
Don't think you need the antialiased here right, it's set globally for the app.
Only exception before was modals as they are outside the render tree.

Reply:
Should I remove it? Don't think it matters that much?

Reply:
Because it is rendered before the already antialiased `MainLayout`, component, I am leaving it as is.
https://github.com/dailydotdev/apps/blob/8ba80c467719c23e4be9b881286ded8821eac161/packages/shared/src/components/MainLayout.tsx#L188